server/chat-commands/core.ts

```diff
@@ -747,7 +747,7 @@ export const commands: ChatCommands = {
 		}
 		user.tempGroup = group;
 		user.updateIdentity();
-		this.sendReply(`|c|~|${this.tr`Your temporary group symbol is now`} \`\`${user.tempGroup}\`\`.`);
+		this.sendReply(`|c|~|/text ${this.tr`Your temporary group symbol is now`} \`\`${user.tempGroup}\`\`.`);
```

Comment:
Should this be fixed inside `sendReply`? I assume so

Reply:
That works, yeah